src/js/components/DataTable/Header.js

```diff
@@ -378,7 +392,7 @@ const Header = forwardRef(
                       justify={align}
                     >
                       {content}
-                      {Icon && <Icon />}
+                      {Icon && <Icon a11yTitle={ariaSort} />}
```

Comment:
instead of `a11yTitle` can we use `aria-label` ?

Reply:
a11yTitle is supported on icons, but aria-label doesn't get passed through: https://github.com/grommet/grommet-icons/blob/master/src/js/icons/index.d.ts#L4

Reply:
Done.